processor/resourcedetectionprocessor/README.md

```diff
@@ -426,6 +426,43 @@ processors:
     override: false
 ```
 
+### Kubeadm Metadata
+
+Queries the K8S API server to retrieve the following resource attributes:
+
+    * k8s.cluster.name
```

Comment:
nit: Shall we link to the `documentation.md` file like what we do in other detectors?